<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParSetFour</class>
 <widget class="QWidget" name="ParSetFour">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>热回收设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="3" colspan="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;热回收机组排风风量(km&lt;span style=&quot; vertical-align:super;&quot;&gt;3&lt;/span&gt;/h)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" rowspan="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>热回收送风机最短开启时间(s)</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" rowspan="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>防冻温度1(°C)</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1" rowspan="2" colspan="2">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_antifreeze1">
          <property name="minimum">
           <double>-100.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>防冻温度3(°C)</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1" colspan="2">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_antifreeze3">
          <property name="minimum">
           <double>-100.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="7" column="3" colspan="3">
         <widget class="QPushButton" name="pushButton_vol_clear">
          <property name="text">
           <string>回收能量清零</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;热回收机组送风风量(km&lt;span style=&quot; vertical-align:super;&quot;&gt;3&lt;/span&gt;/h)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QSpinBox" name="spinBox_in_vol">
          <property name="maximum">
           <number>100000</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1" rowspan="2" colspan="2">
         <widget class="QSpinBox" name="spinBox_min_open_time">
          <property name="maximum">
           <number>10000</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0" rowspan="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>排水电加热启动温度(°C)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" rowspan="2" colspan="2">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_heating_t">
          <property name="minimum">
           <double>-1000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="3" rowspan="2" colspan="2">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>比例阀控制时间间隔(s)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QSpinBox" name="spinBox_out_vol">
          <property name="maximum">
           <number>100000</number>
          </property>
         </widget>
        </item>
        <item row="3" column="3" rowspan="2" colspan="2">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>舍内热回收器数量:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3" rowspan="2" colspan="2">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>防冻温度2(°C)</string>
          </property>
         </widget>
        </item>
        <item row="5" column="5" rowspan="2">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_antifreeze2">
          <property name="minimum">
           <double>-100.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="5" rowspan="2">
         <widget class="QSpinBox" name="spinBox_hoter_count">
          <property name="maximum">
           <number>1000000</number>
          </property>
         </widget>
        </item>
        <item row="1" column="5" rowspan="2">
         <widget class="QSpinBox" name="spinBox_control_time">
          <property name="maximum">
           <number>100000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>304</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>报警设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0" colspan="3">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>舍内温度偏差报警上限(°C)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_deviation_u">
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>20.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>通风级别1-2压力报警上限(Pa)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_pa_1_2_u"/>
        </item>
        <item row="0" column="6">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>通风级别3-16压力报警上限(Pa)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_pa_3_16_u"/>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>舍内温度偏差报警下限(°C)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_deviation_d">
          <property name="minimum">
           <double>-5.000000000000000</double>
          </property>
          <property name="maximum">
           <double>-1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>通风级别1-2压力报警下限(Pa)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_pa_1_2_d"/>
        </item>
        <item row="1" column="6">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>通风级别3-16压力报警下限(Pa)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="7">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_pa_3_16_d"/>
        </item>
        <item row="2" column="4">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>二氧化碳浓度上限(ppm)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QSpinBox" name="spinBox_co2_u">
          <property name="maximum">
           <number>100000</number>
          </property>
         </widget>
        </item>
        <item row="2" column="6">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>报警延时时间(s)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="7">
         <widget class="QSpinBox" name="spinBox_alarm_delay"/>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>舍内湿度报警上限(%)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>舍内湿度报警下限(%)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_humidity_d">
          <property name="maximum">
           <double>80.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_humidity_u">
          <property name="minimum">
           <double>60.000000000000000</double>
          </property>
          <property name="maximum">
           <double>99.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>64</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>安全设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBox_error_swtich_delay"/>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>422</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>热回收机组运行故障后切换为纵向风机的延时时间(s)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
